molgenis-core-ui/package.json

```diff
@@ -24,17 +24,18 @@
     "bootstrap": "3.3.6",
     "bootstrap-fileinput": "4.2.8",
     "brace": "0.5.1",
-    "eonasdan-bootstrap-datetimepicker": "4.15.35",
+    "eonasdan-bootstrap-datetimepicker": "4.17.47",
     "handlebars": "4.0.5",
     "javascript-natural-sort": "0.7.1",
-    "jquery": "2.1.1",
+    "jquery": "3.3.1",
+    "jquery-migrate": "3.0.1",
```

Comment:
Does not work with webpack, see https://github.com/jquery/jquery-migrate/issues/273